Kudu.Services/ServiceHookHandlers/VSOHandler.cs

```diff
@@ -28,68 +38,32 @@ public override DeployAction TryParseDeploymentInfo(HttpRequestBase request, JOb
 
         protected virtual GitDeploymentInfo GetDeploymentInfo(HttpRequestBase request, JObject payload, string targetBranch)
         {
-            // the rest must exist or NullRef.
```

Comment:
Reason for deleting code in this file:

since payload is always static from VSO test notification. there is no point to read the content other than the token.